source/MaterialXRender/Handlers/TinyObjLoader.cpp

```diff
@@ -0,0 +1,190 @@
+#include <MaterialXRender/Handlers/TinyObjLoader.h>
+#include <MaterialXCore/Util.h>
+
+#define TINYOBJLOADER_IMPLEMENTATION
+#include <MaterialXRender/External/TinyObjLoader/tiny_obj_loader.h>
+
+#include <iostream>
+
+namespace MaterialX
+{
+bool TinyObjLoader::load(const std::string& fileName, MeshList& meshList)
```

Comment:
Load code. This is a port from the MaterialXView code which was in it's "Mesh" class. We don't need the tangent code ported since it has already been ported over.
See:
https://github.com/jstone-dev/MaterialX/blob/adsk_contrib/dev/source/MaterialXView/Mesh.cpp